config/testgrids/conformance/conformance-all.yaml

```diff
@@ -157,234 +157,234 @@ dashboards:
 # ppc64le dashboard
 - name: conformance-ppc64le
   dashboard_tab:
-    - name: Periodic ppc64le conformance test on local cluster with containerd as runtime
+    - name: 'Periodic ppc64le conformance test on local cluster with containerd as runtime'
```

Comment:
Why the quotes? They should only be needed if the content of the field might be interpreted as a non-string type and that doesn't seem to be the case to me or am I missing something?

Reply:
there was a unit test faliure error that was happening because of dublicate tab names ... I did this to check if there was an error interpreting these strings... should I remove these?

Reply:
reverted these changes

Reply:
yes please, those are all changes someone has to review

Reply:
done